components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -110,7 +112,7 @@ Configure the OpenSearch `path.repo` setting by SSH to a single OpenSearch serve
 
 ### Backup
 
--   To create the backup, by running the backup command from bastion. The backup command is as shown below:
+- To create the backup, by running the backup command from bastion. The backup command is as shown below:
```

Comment:
how about "Run the backup command from bastion as shown below to create a backup"